primitives/src/constants.rs

```diff
@@ -85,6 +85,18 @@ pub mod chain {
 	/// Core asset id
 	pub const CORE_ASSET_ID: AssetId = 0;
 
+	/// Max fraction of pool to buy in single transaction
+	pub const MAX_OUT_RATIO: u128 = 3;
```

Comment:
these should be preferably constants, but if the XYK use same constants, i would already move the XYK as well